include/constants/regions.h

```diff
@@ -15,6 +15,7 @@ enum Region
     REGION_GALAR,
     REGION_HISUI,
     REGION_PALDEA,
+    REGION_SEVII,
```

Comment:
There's conflicting official information about the Sevii Islands being a region, which is why I didn't add them initially.